front/src/components/Task/TaskBtnGroup.vue

```diff
@@ -62,7 +110,16 @@ export default defineComponent({
   },
 
   computed: {
+    openTaskRoute(): RouteLocationRaw {
+      const params: Record<string, string> = {};
+      params['taskId'] = this.task.id.toString();
+      params['taskSlug'] = this.task.slug;
+      return { name: 'task', params };
+    },
     onItColor() {
+      if (this.dense) {
+        return this.onIt ? 'indigo-10' : 'indigo';
+      }
```

Comment:
Why different colors when it is dense? I think this should not be done since it causes inconsistency with the other views.

Reply:
Because otherwise the _Open task_ and _On it_ buttons would have the same color.

I could change _Open task_ to be a different color (`accent`?) but then it would maybe be a good idea to sync this color with the other views as well.
Or change all _On it_ buttons to use the `indigo` color scheme, but then you would lose some customizability as this is a hard-coded color instead of one of the admin-customizable ones.

Do you have any better suggestions?

Reply:
![image](https://github.com/JJ-8/CTFNote/assets/34778827/a202624e-6428-455a-995c-2216e77e3fb3)
![image2](https://github.com/JJ-8/CTFNote/assets/34778827/dbd0b1a4-078d-48fb-8a81-8c0f23619022)

What do you think of this color scheme? This way it stays consistent for 'on it' across the whole UI. And the color for the 'open task' button is the default 'accent' color which can be configured in the settings.

Reply:
This would correspond with the following diff:
```diff
diff --git a/front/src/components/Task/TaskBtnGroup.vue b/front/src/components/Task/TaskBtnGroup.vue
index e739ca29..d9a40084 100644
--- a/front/src/components/Task/TaskBtnGroup.vue
+++ b/front/src/components/Task/TaskBtnGroup.vue
@@ -11,7 +11,7 @@
         size="sm"
         :title="`Open ${task.title}`"
         icon="edit_note"
-        color="primary"
+        color="accent"
       />
       <q-btn
         v-touch-hold:2000.mouse="handleOnItClick"
@@ -86,9 +86,6 @@ export default defineComponent({
       return { name: 'task', params };
     },
     onItColor() {
-      if (this.group) {
-        return this.onIt ? 'indigo-10' : 'indigo';
-      }
       return this.onIt ? 'secondary' : 'primary';
     },
     onIt(): boolean {
```

Reply:
Looks good!